### ISSUE TITLE
Drag and drop folder from Explorer to Editor area in order to open all files

### ISSUE THREAD

**Issue by djmisterjon** [microsoft/vscode#110155]
Hi, Is it possible to see a drag drop folder for workspace in Vscode?
Or something is preventing this possibility?
![vscode open folder](https://user-images.githubusercontent.com/24865815/98450262-5d611d80-2109-11eb-8e55-f6302fbe7ed0.gif)
 For now i need open file one by one, but inside complexe tree is can be a pain.
I like use one WorkSpace by root folder, and inside the work space , split all subfolder from the root.

**Comment by isidorn** [comment]
This is a fair feature request, however we do not have it on our immediate plan. Thus assigning to backlog and we might add this in the future. Thanks

**Comment by djmisterjon** [comment]
hey thank you, in fact I saw that nobody had made this kind of request, but I feel this need since I discovered that Vscode allows to structure **several workspaces in a project**.
![image](https://user-images.githubusercontent.com/24865815/98599367-d7c8a380-22a9-11eb-8440-d7f385adbfec.png)
This is a unique feature in Vscode and is really great and I can't thank you enough your team for this, 
however it requires a bit of discipline and opening different files for a complex project.
And so the workspace preparation is a bit painful for now if you want to open and prepare the visual structure of a workspace.
It is not a critical thing, but a small luxury for those who like to use this feature and prepare their workspace for their project quickly.

I just wanted to put you in context and the reasons for this request, which I hope will eventually be available.
thanks you for all your work.
@+
